src/main.c

```diff
@@ -132,6 +131,8 @@ int main(int argc, char *argv[]) {
     } else if(rec->count > 0) {
         init_flags.record = 1;
         strncpy(init_flags.rec_file, rec->filename[0], 254);
+    } else if(renderer->count > 0) {
+        strncpy(init_flags.renderer, renderer->sval[0], sizeof(init_flags.renderer));
```

Comment:
This strncpy does not null terminate the output string with overlong --renderer flags
However, please leave this as-is (broken) until we have some saner alternatives to strncpy